x/ecocredit/denom.go

```diff
@@ -0,0 +1,78 @@
+package ecocredit
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+)
+
+// Calculate the ID to use for a new credit class, based on the credit type and
+// sequence number. This format may evolve over time, but will maintain
+// backwards compatibility.
+//
+// The initial version has format:
+// <credit type abbreviation><class seq no>
+func FormatClassID(creditType *CreditType, classSeqNo uint64) (string, error) {
+	return fmt.Sprintf("%s%02d", creditType.Abbreviation, classSeqNo), nil
```

Comment:
```suggestion
	return fmt.Sprintf("%s%03d", creditType.Abbreviation, classSeqNo), nil
```

do we want 3 digits here? ref: https://github.com/regen-network/regen-ledger/pull/413#discussion_r685324116

Reply:
using this formatting, if `classSeqNo` is 1000 for instance then it will just display 1000, if it's equal to 9, then 009 will be displayed, this works as expected

Reply:
oh right, I misread and thought that the `03d` formatting was already there
but aaron said that 2 digits (or 3) was ok for credit class and 3 (or 4 resp.) for credit batches, so let's just keep 2 and 3 which is the case atm